FizzBuzz/spec/fizzbuzz_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "spec_helper"
+require 'FizzBuzzImproved'
+
+describe FizzBuzzImproved do
+  it "returns FizzBuzzSivv if input is 105" do
+    # Given an input of 105
+    # When 3 returns "Fizz", 5 returns "Buzz", and 7 returns "Sivv"
+    # Then return should be "FizzBuzzSivv"
+
+    a = FizzBuzzImproved.new
+    a.set(3, "Fizz")
+    a.set(5, "Buzz")
+    a.set(7, "Sivv")
+    b = a.final(105)
+    b.must_equal "FizzBuzzSivv"
+  end
+
+  it "returns FizzBuzzSivv if input is 105" do
```

Comment:
The titles of these tests need to be updated to indicate what they are actually testing.  This test is actually testing to see if 'GrrMeowFlap' is returned if the input value is 12.